packages/parser/test/shuffle_early_leaver.test.ts

```diff
@@ -0,0 +1,27 @@
+import _ from 'lodash';
+
+import { LoaderResults, loadLogFile } from './testLogLoader';
+
+describe('parsing a log where someone leaves a shuffle match early', () => {
+  const results: LoaderResults = {
+    combats: [],
+    malformedCombats: [],
+    shuffleRounds: [],
+    shuffles: [],
+  };
+
+  beforeAll(() => {
+    const loaded = loadLogFile('shuffle_early_leaver.txt');
+    results.combats = loaded.combats;
+    results.malformedCombats = loaded.malformedCombats;
+    results.shuffleRounds = loaded.shuffleRounds;
+    results.shuffles = loaded.shuffles;
+  });
+
+  it('should return a single match', () => {
```

Comment:
this says single match but seems we're expecting 2 rounds and 0 shuffles. a little confused.. :)